srcalc/src/test/java/gov/va/med/srcalc/vista/RpcVistaPatientDaoTest.java

```diff
@@ -370,4 +386,44 @@ public final void testInvalidDnrNotes() throws Exception
         // This would fail to retrieve DNR notes and the notes would be empty.
         assertEquals(0, patient.getDnrNotes().size());
     }
+    
+    @Test
+    public final void testFullVitals() throws Exception
+    {
+        final VistaProcedureCaller caller = mockVistaProcedureCaller();
+        when(caller.doRpc(
+                RADIOLOGIST_DUZ, RemoteProcedure.GMV_LATEST_VM, String.valueOf(PATIENT_DFN)))
+            .thenReturn(FULL_VITALS);
```

Comment:
The weight 6 months ago code also has no test coverage now. Perhaps you could add that here as well?